### PR TITLE
Update integrations.md

### PR DESCRIPTION
**Pull request by sky-philipalmeida**
Splunk add on for prometheus remote_write.

### CODE REVIEW

File: content/docs/operating/integrations.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ data volumes.
   * [TiKV](https://github.com/bragfoo/TiPrometheus): read and write
   * [VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics): write
   * [Wavefront](https://github.com/wavefrontHQ/prometheus-storage-adapter): write
+  * [Splunk](https://github.com/lukemonahan/splunk_modinput_prometheus#prometheus-remote-write): write
```
Comment (brian-brazil) [inline content/docs/operating/integrations.md]: Keep it sorted

**Comment by brian-brazil** [comment]
Can you add the DCO?

The URL is configurable for remote write, so I'd advise using that over bearer tokens. That way the value won't be hidden on the config page.

**Comment by brian-brazil** [comment]
Thanks!

**Comment by sky-philipalmeida** [comment]
Many thanks

**Pull request merged**